integration-tests/pom.xml

```diff
@@ -25,15 +25,15 @@
 
 	<groupId>com.liferay.ide</groupId>
 	<artifactId>integration-tests</artifactId>
-	<version>3.3.1-SNAPSHOT</version>
+	<version>3.4.0-SNAPSHOT</version>
 
 	<name>Liferay IDE Integration Tests</name>
 
 	<packaging>pom</packaging>
 
 	<properties>
-		<eclipse-site>http://download.eclipse.org/releases/oxygen/201712201001/</eclipse-site>
-		<tycho-packaging-format>'m1'</tycho-packaging-format>
+		<eclipse-site>http://download.eclipse.org/releases/oxygen/201803211000/</eclipse-site>
+		<tycho-packaging-format>'-ga1'</tycho-packaging-format>
```

Comment:
Can we just remove the '-ga1'?  I don't think we need this anymore now that we are gign to release more often.